### DISCUSSION TITLE
bigger area to move columns

### DISCUSSION THREAD

**Discussion by age3rcm** [opentoonz/opentoonz#6458] [Ideas]
### Is your feature request related to a problem? Please describe

i am always frustated when i have to move the order of columns on the list due to the small area to click

### Describe the solution you'd like

bigger area like this:
![OpenToonz_gJBfAMpmUt](https://github.com/opentoonz/opentoonz/assets/36406840/75fda698-8528-42b2-be95-688a61450d8a)

**Comment by RodneyBaker** [comment]
This request might be filled in the short term with adjustment added to Preferences > Interface > Additional Style Sheet.
I'm not sure what that CSS entry would look like however.

Update:  According to @konero this area's values are hardcoded so changes via Preferences would not be an option.
